src/open_dread_rando/files/custom_scenario.lua

```diff
@@ -476,4 +490,27 @@ end
 
 function Scenario.UpdateRoomName(new_subarea)
     RoomNameGui.Update(new_subarea)
+end
+
+function Scenario.UpdateNumTanksMax(num_locs)
+    -- sets Scenario blackboard NumTanksPickedUpMax to num_locs
+    Game.LogWarn(0, "Setting " .. CurrentScenarioID .. ".NumTanksPickedUpMax = " .. num_locs)
+    Scenario.WriteToBlackboard("NumTanksPickedUpMax", "i", num_locs)
+end
+
+function Scenario.IncrementCompletion()
+    -- increment global NumTanksPickedUp
+    local global_itemcount = Scenario.GetBlackboardProp("GAME", "NumTanksPickedUp")
```

Comment:
this function takes an optional default value as its third argument: `local global_itemcount = Scenario.GetBlackboardProp("GAME", "NumTanksPickedUp", 0)`